### ISSUE TITLE
Great program, but when my monitors shut off for sleep but the copper is still on as soon as I press a button to wake my monitors my fans stop and only start again with a full restart of my computer, I have tried everything and can't fix this... any help would be much appreciated.

### ISSUE THREAD

**Issue by BoobyMurphy** [Rem0o/FanControl.Releases#1826]
**Describe the bug**
[A clear and concise description of what the bug is.  Screenshots are worth a thousand words. Hardware support requests do NOT count as a bug. See https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility.]

**Is there a log.txt file next to FanControl.exe with recent date entries?**
[Link it here]

**Relevant hardware specs and setup**
[Laptops and most proprietary pre-built desktops are not supported.]

**Comment by otDan** [comment]
yes this still happens, I commented on a closed issue since I thought it would have been alright to just reopen it, but this is totally broken and unusable if every time my monitors turn off, put it to sleep or hibernate I have to reboot the pc after since the fans are not controlled anymore

**Comment by otDan** [comment]
like the application reports the fan speed changing but I can clearly hear the fans not ramp up at all even if I put the to flat 100% after this

**Comment by otDan** [comment]
also @BoobyMurphy consider modifying the issue to have it correctly commented

**Comment by vitduck** [comment]
@otDan 

Is there any chance that you are using a Gigabyte board ?

**Comment by otDan** [comment]
> I've got an ASUS B550I... similar enough I'd say. Please check in Bios: ErP setting s/b s4/s5, but I'm assuming it is as you said you've got (low) power to start your system with keyboard. Then check your fan curves in Bios... those will control fans until Windows/FanControl take over.

I have no clue what you are referring to, the problem is not that fancontrol doesn't work for a bit of time, the problem is that fan control doesn't work at all after going to sleep, hybernation or after it goes to the monitor off state

**Comment by xlsupport** [comment]
the default install of FanControl runds thru TaskScheduler... and needs a logged-in user. When no user is logged-in there's no contyrol by FanControl. So who/what do you think controls the fans in the meantime? Could it possible be the bios? I hope you got rid of the Asus crapware (AI-Suite/ArmouryCrate).  I suggest Boot into Bios.. check your FanCurves.

**Comment by otDan** [comment]
> xlsupport

I don't care what controls the fans in the meantime as I said the issue is after it wakes up not when it's in sleep or hybernation mode, I don't have any other control software no

**Comment by hcaiano** [comment]
This is also happening to me. If the pc goes to sleep for a while, I can't refresh the sensors unless I fully reboot the system. Is there anything you suggest changing in the BIOS configs (ErP setting s/b s4/s5) to try and fix it? My mobo is MPG X570S CARBON MAX WIFI. Really enjoy the software, but this is a major bug that really needs to be addressed.

**Comment by vitduck** [comment]
> This is also happening to me. If the pc goes to sleep for a while, I can't refresh the sensors unless I fully reboot the system. Is there anything you suggest changing in the BIOS configs (ErP setting s/b s4/s5) to try and fix it? My mobo is MPG X570S CARBON MAX WIFI. Really enjoy the software, but this is a major bug that really needs to be addressed.

When you wake the PC from sleep, and open HWINFO64, do you still see the fans listed under MB's section ? 
If not, then there is likely nothing Fan Control can do since the sensors are gone. 

At least for Gigabyte, some users reported that they fixed it be re-enabling serial port in bios and reflash. 
Otherwise I think you should reach out to MSI support.
